src/qt/coincontroldialog.cpp

```diff
@@ -185,7 +185,7 @@ void CoinControlDialog::buttonSelectAllClicked()
                 ui->treeWidget->topLevelItem(i)->setCheckState(COLUMN_CHECKBOX, state);
     ui->treeWidget->setEnabled(true);
     if (state == Qt::Unchecked)
-        coinControl()->UnSelectAll(); // just to be sure
+        coinControl()->DeSelectAll(); // just to be sure
```

Comment:
Is unselect really not ok? https://en.oxforddictionaries.com/definition/unselect.

Reply:
@jnewbery I found this: https://english.stackexchange.com/questions/18465/unselect-or-deselect

I'm happy to revert this specific change if it is controversial :-)